### PR TITLE
Crowd-Strike Falcon X sandbox

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19771

## Description
new integration for FalconX sandbox,
currently tested locally

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by yaron-libman** [comment]
@ohaim1008 I finished my review, but I would like to make sure that some of the things we discussed yesterday were fixed before providing the docs approved label. Please let me know when you've removed the extra yaml files and replaced the inputs, as we discussed.

**Comment by kirbles19** [comment]
@ohaim1008 Review of the integration and readme is done.

File: Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.yml [hunk 1]
```diff
@@ -0,0 +1,613 @@
...
+      name: system_time
+      required: false
+      secret: false
+    deprecated: false
+    description: Submits a sample SHA256 hash for sandbox analysis.
+    execution: false
+    name: cs-fx-submit-uploaded-file
+    outputs:
+    - contextPath: 'csfalconx.resource.uploaded_id '
+      description: Analysis ID.
```
Comment (bakatzir) [inline Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.yml:161]: you gave both this and the submitted_id the same desc, please re think it.

**Comment by ShahafBenYakir** [comment]
@bakatzir @ohaim1008 Do we an issue about this missing instance?

**Comment by ohaim1008** [comment]
@ShahafBenYakir I opened one now

**Comment by ShahafBenYakir** [comment]
@ohaim1008 Please add the issue number in the conf.json instead of the text (26209), better monitoring this way.

**Pull request merged**